shared-libs/transitions/test/unit/messages.js

```diff
@@ -383,8 +383,8 @@ describe('messages', () => {
 
   it('describe isMessageFromGateway', () => {
     const tests = [
-      ['+774455558888', '77-44-5555-8888', true],
-      ['+774455558889', '77-44-5555-8888', false],
```

Comment:
I think this phone number is actually invalid because when I tried 3rd party verification it failed. Switching over to a valid number fixed it.